lib/jsdom/browser/domtohtml.js

```diff
@@ -8,7 +8,7 @@ exports.domToHtml = function (iterable) {
   let ret = "";
   for (const node of iterable) {
     if (node.nodeType === NODE_TYPE.DOCUMENT_NODE) {
-      ret += parse5.serialize(node, {
+      ret += parse5.serialize(idlUtils.tryWrapperForImpl(node), {
```

Comment:
Sure. domToHtml accepts an iterable of nodes, but before this change we used both wrappers and impls against it. It worked before, but now that certain impls are not array-like (only their Proxy wrappers are) parse5 throws on certain nodes. I'm just making sure only wrappers are passed to parse5.

I also looked into changing the callers of domToHtml to pass wrappers, but for some reason I forgot it seems to be less doable than just changing domToHtml.